src/libs/actions/IOU.js

```diff
@@ -155,29 +164,73 @@ function requestMoney(report, participants, amount, currency, recipientEmail, de
         debtorEmail,
         preferredLocale,
     );
-    const optimisticData = [
+    let optimisticData = [];
+    let successData = [];
+    let failureData = [];
+
+    // First, let's add the data we need just when we are creating a new chat report
+    if (isNewChat) {
+        const optimisticCreateAction = ReportUtils.buildOptimisticCreatedReportAction(recipientEmail);
+        optimisticData = [
+            {
+                onyxMethod: CONST.ONYX.METHOD.SET,
+                key: `${ONYXKEYS.COLLECTION.REPORT}${chatReport.reportID}`,
+                value: {
+                    ...chatReport,
+                    isLoadingReportActions: true,
+                    lastVisitedTimestamp: Date.now(),
+                    pendingFields: {createChat: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD},
+                    isOptimisticReport: true,
```

Comment:
I'm actually not sure. I just saw the chatCreation refactor adds it, so I mimicked it 😅

Reply:
Nvm my comment above, it seems like we introduced it in a recent PR and it seems like we use it for an early return [here](https://github.com/Expensify/App/blob/9a13fa35726cf4a22542074276a96fba338c8234/src/pages/home/report/ReportActionsView.js#L261).

In hindsight, we should probably have checked another property, e.g. `pendingActions`, instead of introducing a new one. Maybe it's worth it to clean this up?